sonaranalyzer-dotnet/its/expected/akka.net/Akka.Tests.Shared.Internals-{E636D23C-3432-4AA9-9A5D-5F29D33D3399}-S100.json

```diff
@@ -15,7 +15,7 @@
 },
 {
 "id":  "S100",
-"message":  "Rename method 'expectMsgPF' to match camel case naming rules, consider using 'ExpectMsgPF'.",
+"message":  "Rename method 'expectMsgPF' to match camel case naming rules, consider using 'ExpectMsgPf'.",
```

Comment:
I thought exactly the same. However, this doesn't raise because of two letter acronym, but because it doesn't start with uppercase letter.